NitroxModel/DataStructures/NitroxVersion.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace NitroxModel.DataStructures;
+
+/// <summary>
+///     Serializable version of <see cref="Version" /> with only major and minor properties.
+/// </summary>
+public readonly struct NitroxVersion : IComparable<NitroxVersion>
+{
+    public int Major { get; init; }
+
+    public int Minor { get; init; }
```

Comment:
Should use `short` here instead of `int` because this is a `struct`, which will be passed around by value.